exercises/resistor-color-duo/README.md

```diff
@@ -21,11 +22,16 @@ The band colors are encoded as follows:
 - Grey: 8
 - White: 9
 
+From the example above:
+brown-green should return 15
+brown-green-violet should return 15 too, ignoring the third color.
```

Comment:
these particular changes to resistor-color-duo deserve their own PR, since they also require a change to the tests. However it is valuable to also see that the changes as-is passed Travis CI.

I suggest that I approve this, but add a commit reversing these specific changes. Then I would squash these commits when merging.

I'd like to give other maintainers about 48 hours to object.

Reply:
Sounds good.